examples/nl-core-SNAQScore-01.xml

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Observation xmlns="http://hl7.org/fhir">
+   <id value="nl-core-SNAQScore-01"/>
+   <meta>
+      <profile value="http://nictiz.nl/fhir/StructureDefinition/nl-core-SNAQScore"/>
+   </meta>
+   <status value="final"/>
+   <code>
+      <coding>
+         <system value="http://snomed.info/sct"/>
+         <code value="8881000146104"/>
+         <display value="SNAQ"/>
```

Comment:
Can we use the full description? (Short Nutritional Assessment Questionnaire)

Reply:
Done (also in SNAQrc and SNAQ65+)!